seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-replace/src/main/java/org/apache/seatunnel/flink/transform/Replace.java

```diff
@@ -0,0 +1,105 @@
+package org.apache.seatunnel.flink.transform;
+
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.types.Row;
+import org.apache.seatunnel.common.config.CheckConfigUtil;
+import org.apache.seatunnel.common.config.CheckResult;
+import org.apache.seatunnel.flink.FlinkEnvironment;
+import org.apache.seatunnel.flink.batch.FlinkBatchTransform;
+import org.apache.seatunnel.flink.stream.FlinkStreamTransform;
+import org.apache.seatunnel.shade.com.typesafe.config.Config;
+
+public class Replace implements FlinkStreamTransform, FlinkBatchTransform {
+
+    private Config config;
+
+    private static final String PATTERN = "pattern";
+    private static final String REPLACEMENT = "replacement";
+    private static final String ISREGEX = "is_regex";
+    private static final String REPLACEFIRST = "replace_first";
+
+    private String pattern = "";
+    private String replacement = "";
+    private Boolean isRegex = false;
+    private Boolean replaceFirst = false;
+
+
+    /**
+     * @param config
+     */
+    @Override
+    public void setConfig(Config config) {
+        this.config = config;
+    }
+
+    /**
+     * @return
```

Comment:
Useless comment. Just remove it will be better.

Reply:
ok ， I have modified